src/util/pointedthing.h

```diff
@@ -61,7 +61,7 @@ struct PointedThing
 	 * Only valid if type is POINTEDTHING_OBJECT.
 	 * The ID of the object the ray hit.
 	 */
-	s16 object_id = -1;
+	u16 object_id = 0;
```

Comment:
is the id actually compared to `0` or `-1` anywhere?

Reply:
i couldn't find any examples of comparing it to anything other than another object_id; https://github.com/minetest/minetest/blob/b3503e7853a52a8c16431f6b983e30c9d25951bc/src/activeobject.h#L131 suggests that "0" is the proper "invalid" value.

Reply:
This looks right. I think the compiler would issue a warning if a `u16` were compared with `-1`.